spiceaidocs/content/en/flight-sql/DBeaver/_index.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
This feels like it should go in quick start, not reference

Reply:
This isn't a reference

Reply:
Agreed